snscrape/modules/instagram.py

```diff
@@ -87,13 +88,22 @@ def _response_to_items(self, response):
 			username = node['node']['owner']['username'] if 'username' in node['node']['owner'] else ''
 			usernameQuery = '?taken-by=' + username
 			cleanUrl = f'https://www.instagram.com/p/{code}/'
+			displayUrl = node['node']['display_url']
+			mediaUrls = []
+			mediaUrls.append(displayUrl)
+			if node['node'].get('edge_sidecar_to_children') != None:
```

Comment:
Does Instagram actually ever return that key as `null`/`None`?
If yes: `if node['node'].get('edge_sidecar_to_children'):`
If no: `if 'edge_sidecar_to_children' in node['node']:`

Reply:
yes, sorted, thank you :+1: